src/dbg/stringformat.cpp

```diff
@@ -21,12 +21,103 @@ enum class StringValueType
     FloatingPointDouble
 };
 
+template<class T> String printFloatValue(FormatValueType value)
+{
+    assert(std::is_same<T, double>() || std::is_same<T, float>());

```

Comment:
This should be static_assert

Reply:
There seems to be a problem with static_assert but I don't use VS2013 anymore.

Reply:
I fixed it, had to use `std::is_same<T, double>::value`